Flow.Launcher.Plugin/SharedCommands/FilesFolders.cs

```diff
@@ -21,7 +21,8 @@ public static class FilesFolders
         /// </summary>
         /// <param name="sourcePath"></param>
         /// <param name="targetPath"></param>
-        public static void CopyAll(this string sourcePath, string targetPath)
+        /// <param name="messageBoxExShow"></param>
+        public static void CopyAll(this string sourcePath, string targetPath, Func<string, MessageBoxResult> messageBoxExShow)
```

Comment:
Can you provide defaults for the methods with the new MessageBox parameter in FilesFolders please.

Reply:
Sure!